tests/test_graphical_units/test_img_and_svg.py

```diff
@@ -0,0 +1,53 @@
+import sys
+from pathlib import Path
+
+import pytest
+from manim import *
+from ..helpers.path_utils import get_project_root
+from ..utils.testing_utils import get_scenes_to_test
+from ..utils.GraphicalUnitTester import GraphicalUnitTester
+
+
+class SVGMobjectTest(Scene):
+    def construct(self):
+        path = (
+            get_project_root()
+            / "tests/test_graphical_units/img_svg_resources/weight.svg"
+        )
+        svg_obj = SVGMobject(str(path))
+        self.add(svg_obj)
+        self.wait()
+
+
+class ImageMobjectTest(Scene):
+    def construct(self):
+        path = (
+            get_project_root()
+            / "tests/test_graphical_units/img_svg_resources/pumpkin1920×1080.png"
+        )
+        imgobj = ImageMobject(str(path))
+        self.add(imgobj)
+        self.wait()
```

Comment:
I think this test is now redundant and can be removed before merging

Reply:
Yes, please, this and the image should be removed. (I am not thrilled about adding a 3MB png to the repository for one single test.)

Reply:
done